src/state/DnDState/DnDAtoms.ts

```diff
@@ -1,10 +1,10 @@
 import { atom } from 'recoil';
 
-interface DnDContainerState {
+export interface DnDContainerState {
```

Comment:
There are two variables exported on this module with the same name, that's A Bad Thing(tm)

Reply:
Changed the names to be different